_layouts/email.html

```diff
@@ -1,69 +1,68 @@
 {% include header.html %}
 
-<div class="content emailPageHeader">
-    <div class="container">
-        <h2>{{ page.name }}</h2>
-        <b>{{ page.city }}, {{ page.state }}</b><br /> {{ site.email_intro_text | markdownify }}
-        <p id="autoopen">{{ site.auto_open_message }}</p>
-        <div class='buttons'>
-            <a href="javascript:void(0);" onclick="openEmail()">Send email</a>
-            <a href="javascript:void(0);" onclick="copyToClipboard(this, `{{ page.permalink }}`, true)">Copy link</a>
-        </div>
-        <p>{{ site.bad_mailto_message }}</p>
-    </div>
-</div>
+<section class="emailPageHeader">
+  <h2>{{ page.name }}</h2>
+  <b>{{ page.city }}, {{ page.state }}</b><br /> {{ site.email_intro_text | markdownify }}
+  <p id="autoopen">{{ site.auto_open_message }}</p>
+  <div class='buttons'>
+    <a href="javascript:void(0);" onclick="openEmail()">Send email</a>
+    <a href="javascript:void(0);" onclick="copyToClipboard(this, `{{ page.permalink }}`, true)">Copy link</a>
+  </div>
+  <p>{{ site.bad_mailto_message }}</p>
+</section>
 
-<div class="content emailContentSection">
-    <div class="container">
-        <div class="emailContent">
-            <div class="recipients">
-                <b>To:</b> <span class="copyToClipboard" onclick="copyToClipboard(this, `{{ page.recipients | join: ', ' }}`)">🔗</span>
-                {{ page.recipients | join: ', ' }}
-            </div>
-            {% if page.cc %}
-            <div class="recipients">
-                <b>CC:</b> <span class="copyToClipboard" onclick="copyToClipboard(this, `{{ page.cc | join: ', ' }}`)">🔗</span>
-                {{ page.cc | join: ', ' }}
-            </div>
-            {% endif %}
-            <div class="recipients">
-                <b>Subject:</b> {{ site.default_subject_line }}
-            </div>
-            <div>
-                <b>Message:</b> <i>(Don't forget to replace the [x]'s with your information!)</i>
-                <span class="copyToClipboard" onclick="copyToClipboard(this, `{{ page.body }}`)">🔗</span>
-                <br /> {{ page.body | markdownify }}
-            </div>
-        </div>
+<article class="emailContentSection">
```

Comment:
semantic element 😍😍😍

Reply:
always, though it's debatable if this is article worthy...